std_msgs/String pointcloud_topic
std_msgs/String image_topic

---
sensor_msgs/PointCloud2 filtered